.github/workflows/conformance-kind-proxy-embedded.yaml

```diff
@@ -118,6 +118,24 @@ jobs:
           cilium connectivity test ${{ steps.vars.outputs.connectivity_test_defaults }} \
           --junit-file "cilium-junits/${{ env.job_name }}.xml" --junit-property github_job_step="Run connectivity test"
 
+      - name: Install Cilium with secret sync disabled
+        id: install-cilium-secret-sync-disabled
+        run: |
+          helm upgrade cilium ./install/kubernetes/cilium \
+            --namespace kube-system \
+            --reuse-values \
+            --set=tls.secretSync.enabled=false
+
+          kubectl -n kube-system rollout restart ds/cilium deployment/cilium-operator
+          cilium status --wait --wait-duration=10m
+          kubectl -n kube-system get pods
+
+      - name: Run L7 related connectivity test
+        run: |
+          cilium connectivity test --test="l7|sni|check-log-errors" \
+            --junit-file "cilium-junits/${{ env.job_name }}-secret-sync.xml" \
+            --junit-property github_job_step="Run connectivity test with secret sync disabled"
+
       - name: Features tested
```

Comment:
Please copy this step and put it on line 120. Then change line 143 to be `json-filename: "${{ env.job_name }} - with secret sync"`.

This way we are capturing the coverage for both connectivity test cases.